websites/S/Sonic Retro/presence.ts

```diff
@@ -0,0 +1,285 @@
+const presence = new Presence({
+		clientId: "970743721404530798"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, search, hostname } = window.location;
+	switch (hostname) {
+		case "sonicretro.org": {
+			presenceData.smallImageKey = "home_512";
+			if (search) {
+				presenceData.largeImageKey = "search";
+				if (new URLSearchParams(document.location.search).get("s") === "")
```

Comment:
Also just in case this is used elsewhere in the future I'd recommend declaring the search params variable outside the switch statement along with the others
Also not sure if the parenthesis are necessary, remove them if prettier complains
```suggestion
				if (!(new URLSearchParams(search).get("s")))
```